demo/src/javascripts/components/page.js

```diff
@@ -77,28 +79,28 @@ class Page extends Component {
     const {tabs, mapFocus} = this.state;
     const activeTab = location.query.tab || Object.keys(tabs)[0];
 
-    return Object.keys(tabs).map(tabName => {
-      const tab = tabs[tabName];
+    return Object.keys(tabs).map(_tabName => {
+      const _tab = tabs[_tabName];
```

Comment:
Nit: I see the problem, but underscore always looks awkward. Maybe `selectedTabName` and `selectedTab`, or `activeTabName` at the top would look better.